### ISSUE TITLE
Add note to key

### ISSUE THREAD

**Issue by TheLastProject** [keys-pub/app#13]
Someone sent me an email and I imported there key, but there is no way to add a note or anything so my list of keys only shows the key and an empty field for User. Once this becomes 2 or 3 such externally imported keys without user info, I am not going to be able to tell who any of the keys belong to.

It would be very useful to be able to add a note or something to a key. Any suggestions as to how you'd prefer this to work? I'm willing to work on this myself.

**Comment by gabriel** [comment]
Yeah this would be a good feature. It would probably work by adding a note and label field to the keyring item and a way to add notes and labels from the app.
